targets/TARGET_STM/TARGET_STM32H7/TARGET_STM32H747xI/TARGET_PORTENTA_H7_M7/TOOLCHAIN_GCC_ARM/STM32H747xI.ld

```diff
@@ -33,11 +33,12 @@ MEMORY
   RAM (xrw)                     : ORIGIN = RAM_START + 0x298, LENGTH = RAM_SIZE - 0x298
   RAM_D2 (xrw)                  : ORIGIN = 0x30000000, LENGTH = 288K
   OPENAMP_RSC_TAB (rwx)         : ORIGIN = 0x38000000, LENGTH = 1K
-  OPEN_AMP_SHMEM (rwx)          : ORIGIN = 0x38000400, LENGTH = 63K
+  OPEN_AMP_SHMEM (rwx)          : ORIGIN = 0x38000400, LENGTH = 62K
+  PDM_MEM (xrw)                 : ORIGIN = 0x3800FC00, LENGTH = 1k
   FLASH (rx)                    : ORIGIN = MBED_APP_START, LENGTH = MBED_APP_SIZE
 }
  __OPENAMP_region_start__  = ORIGIN(OPEN_AMP_SHMEM);
- __OPENAMP_region_end__ = ORIGIN(OPEN_AMP_SHMEM) + LENGTH(OPEN_AMP_SHMEM);
+ __OPENAMP_region_end__ = ORIGIN(OPEN_AMP_SHMEM) + LENGTH(OPEN_AMP_SHMEM) - 1;
```

Comment:
Is the `-1` needed beacuse in openamp we write something in `__OPENAMP_region_end__` location? In this case it may be better to patch openamp itself :slightly_smiling_face:

Reply:
No, i've changed because i've thought `__OPENAMP_region_end__` should be `0x3800FBFF` and not `0x3800FC00`, but then double checking here https://github.com/arduino/ArduinoCore-mbed/blob/66f1f03f27dbf97a79a906e823d1f81a9cc881a7/libraries/openamp_arduino/src/openamp_conf.h#L152 i've found out is used to calculate `SHM_SIZE` so i'll revert the change.